platform/jvm/gradle-test-app/src/main/java/io/bitdrift/gradletestapp/FirstFragment.kt

```diff
@@ -147,7 +147,6 @@ class FirstFragment : Fragment() {
 
     private fun provideOkHttpClient(): OkHttpClient {
         return OkHttpClient.Builder()
-            .eventListenerFactory(CaptureOkHttpEventListenerFactory())
```

Comment:
If there are multiple okhttp clients will it affect them all? e.g. if they use a separate one for apollo graphql

Reply:
Yeah the plugin will traverse all the clients in the system and add it everywhere. If using both at the same time causes issues we should figure out how to make the interoperate well, this is probably easier than trying to figure out how to make it conditional

Reply:
ok i'm working on that pr now, i'll think about it